src/Missings.jl

```diff
@@ -207,4 +208,262 @@ missing
 """
 passmissing(f) = PassMissing{Core.Typeof(f)}(f)
 
+
+"""
+   skipmissings(args...)
+
+Return a tuple of iterators wrapping each of the iterators in `args`, but
+skipping elements at positions where at least one of the iterators returns `missing`.
+
+# Examples
+```
+julia> x = [1, 2, missing, 4]; y = [1, 2, 3, missing];
+
+julia> tx, ty = skipmissings(x, y)
+(SkipMissings{Array{Union{Missing, Int64},1},Tuple{Array{Union{Missing, Int64},1}}}(Union{Mi
+ssing, Int64}[1, 2, missing, 4], (Union{Missing, Int64}[1, 2, 3, missing],)), SkipMissings{A
+rray{Union{Missing, Int64},1},Tuple{Array{Union{Missing, Int64},1}}}(Union{Missing, Int64}[1
+, 2, 3, missing], (Union{Missing, Int64}[1, 2, missing, 4],)))
+
+julia> collect(tx)
+2-element Array{Any,1}:
```

Comment:
Does this really produce an `Array{Any}`? If so that's a bug. Can you add a test for that?

Reply:
It infers correctly, thankfully.

Reply:
But it shouldn't produce an `Array{Any}`. Why not an `Array{Int}`?